commons/protocol/src/main/scala/sbt/protocol/Keys.scala

```diff
@@ -82,11 +82,29 @@ object KeyList {
   implicit val pickler: Pickler[KeyList] = genPickler[KeyList]
 }
 
+// TODO - This is a bad name...
+/** The notion of classpath dependencies between projects. */
+final case class ClasspathDep(
+  project: ProjectReference,
+  configuration: Option[String])
+object ClasspathDep {
+  implicit val picklerUnpickler = PicklerUnpickler.generate[ClasspathDep]
+}
+/** The notiion of build dependencies between projects. */
+final case class ProjectDependencies(
+  classpath: Seq[ClasspathDep],
+  aggregate: Seq[ProjectReference])
+object ProjectDependencies {
+  implicit val picklerUnpickler = PicklerUnpickler.generate[ProjectDependencies]
+}
+
 /** Core information returned about projects for build clients. */
 final case class MinimalProjectStructure(
   id: ProjectReference,
   // Class names of plugins used by this project.
-  plugins: Vector[String])
+  plugins: Vector[String],
+  /** A possible list of dependencies between projects..   Added in 0.2 of the protocol. */
```

Comment:
SemanticVersioning 2.0.0 Article 7:

> Minor version Y (x.Y.z | x > 0) MUST be incremented if new, backwards compatible functionality is introduced to the public API.

So we need to go 1.1. This allows us to add new batch of oneway trip files to make sure 1.0 input can be parsed using current 1.1 code, and vice versa.